### PR TITLE
RALLY-007 Globe scroll to top

### PR DESCRIPTION
**Pull request by rally-adam**
I'm very much on the fence about this; interested in other opinions. Would be perfectly happy to abandon for now

### CODE REVIEW

**Comment by ryan-ludwig** [comment]
I this punting on this until design has had a chance to test things on their devices would be good.

**Comment by rally-adam** [comment]
@ryan-ludwig 👍

**Comment by mcstover** [comment]
@rally-adam Could you please merge the latest from master into this PR? Seems like some of this work is already there...

**Comment by rally-adam** [comment]
@mcstover woops! should be good now. I do tend to agree w/ Ryan that we should probably table this for more testing / vetting w/ design

File: src/components/15Years/15YearsGlobe.vue [hunk 1]
```diff
@@ -104,13 +104,21 @@ export default {
...
 				this.gkview.animateToLatLon(result.lat + offset, result.lon);
 				const callout = new CalloutDefinition(result.lat, result.lon, PinCallout, result.properties);
 				callout.altitude = 0.035;
 				this.calloutManager.replaceCallouts([...this.callouts, callout]);
 				this.$emit('selectcountry', result.properties);
 				this.currentCallout = callout;
+
+				if (isMobileLayout) {
+					console.log(window.scrollY, window.innerHeight);
+					if (window.scrollY > window.innerHeight * 0.45) {
```
Comment (mcstover) [inline src/components/15Years/15YearsGlobe.vue:118]: @emuvente This check is a good rough estimate but should get us moving forward. We might later like to just add a check to see if the Country element is in the viewport.

**Review by mcstover — APPROVED** [review]

File: src/components/15Years/15YearsGlobe.vue (hunk 1 above)
Comment (emuvente) [reply src/components/15Years/15YearsGlobe.vue:118]: agreed

**Review by emuvente — APPROVED** [review]

**Review by ryan-ludwig — APPROVED** [review]

**Comment by mcstover** [comment]
Going to merge this in which will publish it to Storybook we can review on mobile from there.

**Pull request merged**